source/python.js

```diff
@@ -5574,7 +5574,7 @@ python.Unpickler.BinaryReader = class {
     float64() {
         const position = this._position;
         this.skip(8);
-        return this._view.getFloat64(position, true);
+        return this._view.getFloat64(position, false);
```

Comment:
It seems that pickle save float object in big-endian, https://github.com/python/cpython/blob/a0c7df09689769f3016dc9145eb56341aac631a9/Lib/pickle.py#L784